src/Microsoft.AspNetCore.Mvc.Core/ModelBinding/Binders/ComplexTypeModelBinder.cs

```diff
@@ -47,6 +48,17 @@ public Task BindModelAsync(ModelBindingContext bindingContext)
                 return TaskCache.CompletedTask;
             }
 
+            // The following check causes the ComplexTypeModelBinder to NOT participate in binding structs as 
+            // reflection does not provide information about the implicit parameterless constructor for a struct.
```

Comment:
Add back the bit about an inability to create non-`null` instances of `Nullable<T>` since that makes this "restriction" minor.